README.md

```diff
@@ -462,13 +462,29 @@ require'nvim-treesitter.configs'.setup {
 
 Tree-sitter based folding. _(Technically not a module because it's per windows and not per buffer.)_
 
+```lua
+vim.o.foldmethod = "expr"
+vim.o.foldexpr = "v:lua.vim.treesitter.foldexpr()"
+-- With the above settings applied, vim will fold everything by default when
+-- opening a file... setting foldlevelstart to a high value allows you to
+-- start with no folds and then toggle them as desired with the za keymap.
+vim.o.foldlevelstart = 99
+```
+
+or if you prefer vimscript:
+
 ```vim
 set foldmethod=expr
-set foldexpr=nvim_treesitter#foldexpr()
-set nofoldenable                     " Disable folding at startup.
+set foldexpr=v:lua.vim.treesitter.foldexpr()
```

Comment:
Built-in foldexpr seems available from neovim 0.9+ and ported from this repo (https://github.com/neovim/neovim/pull/22299). Does this mean that `nvim_treesitter#foldexpr()` is deprecated and no longer used?

Reply:
That is correct. (I mentioned that the README is a mess, didn't I?)